lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Task/TaskRuntime.cs

```diff
@@ -56,43 +71,36 @@ public TaskRuntime(IInjector taskInjector, string contextId, string taskId, Hear
 
                 // do not rethrow since this is benign
             }
+
             try
             {
-                heartBeatManager.EvaluatorSettings.NameClient = _injector.GetInstance<INameClient>();
```

Comment:
I know this is deprecated apparoach. If a code path reaches to the old approach, where the NameCLient will be set if this code is removed?